### PR TITLE
Add io.github.julian_hochhaus.LG4X_V2

### PR DESCRIPTION
**Pull request by kevinsmia1939**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:https://github.com/Julian-Hochhaus/LG4X-V2/issues/78#issuecomment-1643431841** 
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission

Please add https://github.com/Julian-Hochhaus as a maintainer as well.

### CODE REVIEW

File: io.github.julian_hochhaus.LG4X_V2.yaml [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+cleanup:
+  - /include
+  - /share/man
+modules:
+  - name: meson
```
Comment (hfiguiere) [inline io.github.julian_hochhaus.LG4X_V2.yaml:17]: why do you need to build meson?

Comment (kevinsmia1939) [reply io.github.julian_hochhaus.LG4X_V2.yaml:17]: SciPy Python library version 1.11.1 need meson >=1.1.0, the Sdk provide 0.63.3.

**Comment by kevinsmia1939** [comment]
bot, build io.github.julian_hochhaus.LG4X_V2

**Comment by kevinsmia1939** [comment]
bot, build io.github.julian_hochhaus.LG4X_V2

**Comment by kevinsmia1939** [comment]
I think it is ready.

**Comment by kevinsmia1939** [comment]
Please merge with https://github.com/Julian-Hochhaus  as co-maintainer.

**Pull request closed without merging**